addons/mail/static/src/scss/mail_activity.scss

```diff
@@ -162,18 +162,27 @@
 
         .o_activity_link_kanban {
             visibility: hidden;
+            @include media-breakpoint-down(sm) {
+                visibility: visible;
+            }
         }
 
-        ul.o_activity {
+        div.o_activity {
             width: 290px;
             padding: 0px 0px;
 
-            & > li:last-child {
+            & > div:last-child {
                 background-color: $o-brand-lightsecondary;
                 color: gray('900');
                 line-height: 3em;
             }
-            li > ul.o_activity_log {
+            & > .dropdown-item {
+                padding: 0px;
+                &:hover {
```

Comment:
Why make the background transparent on hover for **_"+ Schedule an activity"_**?
This was not the case before the migration to bootstrap v4.

Apparently, this is to avoid having a dark background color when hovering on the title of the dropdown item. However, the footer is transparent on hover, which I suppose this is not the intended behaviour: https://drive.google.com/file/d/19zVSm92V9L-QusQ9pK-42rYI9J7FWWCr/view?usp=sharing

Reply:
Hello, It's fixed, you can check now. Thanks.